src/init/command.template

```diff
@@ -0,0 +1,11 @@
+# This is the entrypoint for vmtest commands in both kernel and image targets.
+# We user a small rendering engine so it's easier to read/write more complex logic.
+
+# Kernel targets share host FS so this `cd` should succeed. If it doesn't,
+# the error is shown to the user as a hint.
```

Comment:
This is only a *should* since #34 :) . It could be solveable if needed though.

Reply:
Yeah, good catch. Seems like the fix should be to disable `cd ...` logic if custom rootfs is provided. Since cwd on host is irrelevant w.r.t. provided rootfs

Reply:
Went ahead and added the proper fix

Reply:
IIRC, the issue was that we `cd` into the host path. But a simple workaround would be to `cd` into `/mnt/vmtest` right? So maybe substituting host_shared to SHARED_9P_FS_MOUNT_PATH may do the trick?

Reply:
I don't think `SHARED_9P_FS_MOUNT_PATH` works for relative paths, right? So a command like `ls ../..` would still break.

Reply:
oh yeah, I see: https://github.com/danobi/vmtest/blob/5b498621f32c9175c6d2566d19a7a7ccef43dc90/src/vmtest.rs#L82

Definitely, relative path won't work. We would be in the same directory.... but under a different mount point.